wbstack/src/Info/GlobalSet.php

```diff
@@ -109,13 +110,14 @@ private static function cacheKey( $requestDomain ) {
      */
     private static function getInfoFromApi( $requestDomain ) {
         // START generic getting of wiki info from domain
-        $url = 'http://' . getenv( 'PLATFORM_API_BACKEND_HOST' ) . '/backend/wiki/getWikiForDomain?domain=' . urlencode($requestDomain);
+        $url = 'http://' . getenv( 'PLATFORM_API_BACKEND_HOST' ) . '/backend/wiki/getWikiForDomain/?domain=' . urlencode($requestDomain);
```

Comment:
Adding this `/` in the production deployment and image causes the request to result in a redirect.

```
65c5dd4767-5f58k:/var/www/html# curl api-app-backend.default.svc.cluster.local/backend/wiki/getWikiForDomain/?domain=addshore-alpha.wiki.opencura.com
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="http://api-app-backend.default.svc.cluster.local/backend/wiki/getWikiForDomain?domain=addshore-alpha.wiki.opencura.com">here</a>.</p>
<hr>
<address>Apache/2.4.38 (Debian) Server at api-app-backend.default.svc.cluster.local Port 80</address>
</body></html>
```

So either we need to:
 - Make these requests follow redirects
 - Switch the dev stuff to not redirect from no `/` to with a `/`

Reply:
done with redirect